py/test/selenium/webdriver/marionette/mn_launcher_tests.py

```diff
@@ -31,11 +33,8 @@ def test_we_can_launch_multiple_firefox_instances(capabilities):
     driver3.quit()
 
 
-def test_launch_firefox_with_none_service_log_path(capabilities):
-    driver = Firefox(capabilities=capabilities, service_log_path=None)
-    driver.quit()
-
-
-def test_launch_firefox_with_empty_string_service_log_path(capabilities):
-    driver = Firefox(capabilities=capabilities, service_log_path="")
-    driver.quit()
+@pytest.mark.parametrize('log_path', [None, "", ("geckodriver.log", "w")])
+def test_launch_firefox_with_service_log_path(log_path):
```

Comment:
can we inspect the underlying `log_file` attr for assertion? (calling `.quit()` is unnecessary also)